jina/hubble/helper.py

```diff
@@ -0,0 +1,59 @@
+"""Module for helper functions for Hub API."""
+
+import io
+from pathlib import Path
+
+
+def archive_package(package_folder: 'Path') -> 'io.BytesIO':
+    """
+    Archives the given folder in zip format and return a data stream.
+    :param package_folder: the folder path of the package
+    :return: the data stream of zip content
+    """
+    import os
+    import zipfile
```

Comment:
standard python library can be imported at header, as they are usually very fast to load